hawkmoth/docstring.py

```diff
@@ -142,7 +142,14 @@ def get_docstring(self, transform=None):
 
         text = Docstring._nest(text, self._indent)
 
-        args = ', '.join(self._args) if self._args is not None else None
+        name = self._get_decl_name()
```

Comment:
I guess you meant to use name later, but didn't?

Reply:
Ah, it was for consistency and readability, I wanted to have a smaller name to pass to the function. But when I decided the commits made more sense in the reverse order, I clearly messed up this part of the rebase.

I'll probably need to force push to resolve it.